src/server/generic_family.cc

```diff
@@ -1514,28 +1517,29 @@ OpStatus GenericFamily::OpMove(const OpArgs& op_args, string_view key, DbIndex t
   auto& db_slice = op_args.shard->db_slice();
 
   // Fetch value at key in current db.
-  auto [from_it, from_expire] = db_slice.FindExt(op_args.db_cntx, key);
-  if (!IsValid(from_it))
+  auto from_res = db_slice.FindMutable(op_args.db_cntx, key);
+  if (!IsValid(from_res.it))
     return OpStatus::KEY_NOTFOUND;
 
   // Fetch value at key in target db.
   DbContext target_cntx = op_args.db_cntx;
   target_cntx.db_index = target_db;
-  auto [to_it, _] = db_slice.FindExt(target_cntx, key);
-  if (IsValid(to_it))
+  auto to_res = db_slice.FindMutable(target_cntx, key);
```

Comment:
I think this should be ReadOnly we just check the to_res is valid

Reply:
Nice catch, done.